src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -237,23 +194,27 @@ public void openBrowserIntoPanel(String uri, ContentManager contentManager, Stri
       devToolsInstance.openPanel(project, uri, contentManager, tabName, pageName);
     }
     else {
-      @Nullable final OSProcessHandler handler =
-        isBazel(project) ? getProcessHandlerForBazel() : getProcessHandlerForPub();
+      if (isBazel(project)) {
+        try {
+          devToolsInstance = getDevToolsInstance().get(15, TimeUnit.SECONDS);
```

Comment:
why is there a 15 second timeout?

Reply:
I can't remember what we decided on this - keep it? Otherwise it could wait indefinitely without raising an exception.